routers/web/repo/view.go

```diff
@@ -348,6 +350,16 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 		if issueConfigErr != nil {
 			ctx.Data["FileError"] = strings.TrimSpace(issueConfigErr.Error())
 		}
+	} else if actions.IsWorkFlow(ctx.Repo.TreePath) {
+		content, err := actions.GetContentFromEntry(entry)
+		if err != nil {
+			ctx.Error(http.StatusInternalServerError, err.Error())
+			return
+		}
```

Comment:
I guess we can just ignore (log.Error) this error, there are too many 500 errors in Gitea, most of them could be avoided.